keras_nlp/models/opt/opt_presets.py

```diff
@@ -24,6 +24,7 @@
             "params": 125237760,
             "official_name": "OPT",
             "path": "opt",
+            "model_card": "https://ar5iv.labs.arxiv.org/html/2205.01068#:~:text=D.1-,Model%20Details,-%E2%80%A2%20Person%20or",
```

Comment:
Maybe we should link https://github.com/facebookresearch/metaseq/blob/main/projects/OPT/model_card.md ? It's a link to a link, but points to the right appendix in the paper (and hopefully would be something the metaseq team would update if ever they needed to).